obc/app/tools/interface_debug_tool/source/main.c

```diff
@@ -23,6 +23,7 @@
 #include <i2c.h>
 
 #include <string.h>
+#include <test_rtc.h>
```

Comment:
To follow our naming convention as this is a local include:
```suggestion
#include "test_rtc.h"
```
Also this should be in line with the other tests so I would move this after line 16